debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/wal2json/Wal2JsonReplicationMessage.java

```diff
@@ -96,6 +101,10 @@ public long getTransactionId() {
 
     @Override
     public String getTable() {
+        if(rawMessage == null) {
+            return null;
+        }
```

Comment:
I'm not sure what the behavior should be here. The null thing does not seem quite "java like," I just did this so the code change in RecordsStreamProducer was cleaner.

Reply:
Probably best to revamp it to return `Optional<String>`.